docs/docs/frigate/installation.md

```diff
@@ -237,3 +237,26 @@ It is recommended to run Frigate in LXC for maximum performance. See [this discu
 ## ESX
 
 For details on running Frigate under ESX, see details [here](https://github.com/blakeblackshear/frigate/issues/305).
+
+## Synology NAS on DSM 7
+you can run Frigate NVR on DSM 7.x. tested on DSM 7.1.1-42962 Update 4
+
+these were the settings configured (no tpu device just using the cpu from the device which is not great but still works 100%):
```

Comment:
```suggestion
```

Reply:
We absolutely recommend a coral or other detector in every circumstance.

I'm concerned about this too since device passthrough is usually the big problem with Synology deployments